dev-riscv/scripts/configure.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+OPTIND=1
+
+variants=core
+level=release
+
+while getopts "hv:l:" opt; do
+    case "$opt" in
+    h)
+        echo "usage: configure.sh [-h] [-v jvm-variants] [-l debug-level]"
+        echo "       -h show help"
+        echo "       -v set --with-jvm-variants flag (server, client, minimal, core, zero, zeroshark, custom). default is core"
+        echo "       -l set --with-debug-level flag (release, fastdebug, slowdebug, optimized). default is release"
+        exit 0
+        ;;
+    v)  variants=$OPTARG
+        ;;
+    l)  level=$OPTARG
+        ;;
+    esac
+done
+
+bash configure \
+    --with-jvm-variants=$variants \
+    --with-debug-level=$level \
+    --with-native-debug-symbols=internal \
```

Comment:
The debug flags are taken from here: https://hg.openjdk.java.net/jdk-updates/jdk9u/raw-file/tip/common/doc/building.html#common-configure-arguments

With these flags gdb server **does not** show the following error:
```
No debugging symbols found in build/linux-riscv64-core-fastdebug/jdk/bin/java
```

However, the client shows the following:
```
No symbol table is loaded.  Use the "file" command.
```